lib/model/survey_details_model.dart

```diff
@@ -2,20 +2,25 @@ import 'package:equatable/equatable.dart';
 import 'package:flutter_survey/api/response/question_response.dart';
 import 'package:flutter_survey/api/response/survey_details_response.dart';
 import 'package:flutter_survey/model/question_model.dart';
+import 'package:collection/collection.dart';
 
 class SurveyDetailsModel extends Equatable {
   final String id;
   final String title;
   final String description;
   final String coverImageUrl;
   final List<QuestionModel> questions;
+  final String intro;
+  final String outro;
 
   const SurveyDetailsModel({
     required this.id,
     required this.title,
     required this.description,
     required this.coverImageUrl,
     required this.questions,
+    required this.intro,
+    required this.outro,
```

Comment:
I'm not sure we need them for what? 💭 As I understand correctly, they're `question type`, does it make sense to include the question type values as fields in the model?

Reply:
@luongvo We're using this as the message for the thank you screen 🙏 

However, we could potentially remove `intro` as we're not using it 💭 

Please correct me if I'm wrong @Shayokh144

Reply:
@Tuubz we can remove intro as it is not used

Reply:
Or... we could rename `outro` to `thankYouMessage` to avoid confusion